docker-compose.yml

```diff
@@ -129,5 +129,30 @@ services:
                 aliases:
                     - mongo-useradm
 
+    mender-conductor:
```

Comment:
imo not a good idea. conductor is both mandatory and not swappable, so it has to be there always as is.